demo/index.js

```diff
@@ -30,18 +29,38 @@ controls.autoRotateSpeed = 4;
 const nonOptimizedGltfBasePath = "assets/";
 
 const lods = [
-  { basePath: "assets/duck", name: "duck", position: [30, 0, 0] },
-  { basePath: "assets/airplane", name: "airplane", position: [-30, 0, 0] },
-  { basePath: "assets/skull", name: "skull", position: [0, 0, 30] },
-  { basePath: "assets/dragon", name: "dragon", position: [0, 0, -30] },
+  {
+    relativePathToLodeOutputFolder: "./lode-build",
+    basePath: "assets/duck",
+    name: "duck",
```

Comment:
looks like we could use some "convention over configuration" here?
`relativePathToLodeOutputFolder` should probably be part of basic "lode config" object?
basePath should be combination of "lode-config-base-path" and "name"?